src/main/resources/templates/newarcadechallenge.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head th:include="base :: header"></head>
+<body>
+<nav th:include="navbar :: navigation"></nav>
+<div class="container">
+    <h1>New challenge</h1>
+    <div class="row">
+        <form th:action="@{/createArcadeChallenge}" method="post">
+            <label>challenge name:</label><br />
+            <input type="text" name="challengeName" size="50"/><br />
+            <label>challenge description</label><br />
+            <textarea rows="10" cols="80" name="challengeDesc"></textarea><br />
+            <label>Challenge type:</label>
+            <label>Realm:</label>
+            <select name="realm">
+                <option value="BEGINNER">Beginner</option>
+                <option value="OBJECTORIENTED">Object-oriented</option>
+                <option value="DATASTRUCTURES">Data structires</option>
+            </select><br />
```

Comment:
Should be easy to implement properly using `th:each` here and `Realm.values()` in the controller.